lib/backend/etcdbk/etcd_test.go

```diff
@@ -366,3 +367,31 @@ func (s *EtcdSuite) TestSyncLegacyPrefix(c *check.C) {
 	})
 	s.reset(c)
 }
+
+// CompareAndSwapIssue4786 ensures that the backend reacts with a proper
+// error message if client sends a message exceeding the configured size maximum
+// See https://github.com/gravitational/teleport/issues/4786
+func (s *EtcdSuite) TestCompareAndSwapOversizedValue(c *check.C) {
```

Comment:
Please use the https://github.com/gravitational/teleport/blob/master/rfd/0001-testing-guidelines.md style for tests

Reply:
Can you elaborate? Should the test use the `testing` package directly?

Reply:
Yep, use `testing` instead of `gopkg.in/check` and `testify` for aserts